packages/terra-table/docs/guides/SectionWithinSection.md

```diff
@@ -0,0 +1,144 @@
+# Terra Table - Sections with Subsections
+
+With the inclusion of sections in the table, there are two recommended patterns for created nested collapsible sections. First is collapsible sections and static subsections, and the second is static sections and collapsible subsections. The guidance from UX is to not have both collapsible sections and collapsible subsections in a table.
+
+## State Management
+As section and subsection have the same API, we'll be walking through the expectation of a collapsible section in only one pattern.
```

Comment:
Should we call this Doc page __guide/TableSectionPattern__?  I know this example is detailed for section within subsection but it sounds like it is also explaining the basic usage pattern for both section & subsection. If not should we add a reference in their READMES indicating they follow the same selection pattern implantation?

Reply:
I'll add an additional pure section guide to cover some of these concepts.

Reply:
Updated.